libsolidity/parsing/Parser.cpp

```diff
@@ -1061,14 +1061,27 @@ ASTPointer<IdentifierPath> Parser::parseIdentifierPath()
 	RecursionGuard recursionGuard(*this);
 	ASTNodeFactory nodeFactory(*this);
 	nodeFactory.markEndPosition();
+
+	SourceLocation identifierLocation{currentLocation()};
```

Comment:
```suggestion
	SourceLocation identifierLocation = currentLocation();
```
I personally like the `{}`-initialization, but I think that would break style (here) otherwise.